backend/app/api/ws/chat_namespace.py

```
@@ -0,0 +1,1204 @@
+                    final_message = attachment_service.build_message_with_attachment(
+                        message, attachment
+                    )
+
+            # Prepare tools
+            tools = None
+            if payload.enable_web_search and settings.WEB_SEARCH_ENABLED:
+                from app.services.chat.tools import get_web_search_tool
+
+                web_search_tool = get_web_search_tool()
+                if web_search_tool:
+                    tools = [web_search_tool]
```

Ruff (0.14.8)

Local variable `tools` is assigned to but never used

Remove assignment to unused variable `tools`

(F841)